Dockerfile.cloud

```diff
@@ -1,31 +1,40 @@
-# Build stage
+###### [STAGE] Build ######
 FROM node:18-alpine as builder
 WORKDIR /etc/logto
 ENV CI=true
 
-# No need for build
+# No need for Docker build
 ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true
 
-# Install toolchain
+### Install toolchain ###
 RUN npm add --location=global pnpm@^7.14.0
 # https://github.com/nodejs/docker-node/blob/main/docs/BestPractices.md#node-gyp-alpine
 RUN apk add --no-cache python3 make g++
 
 COPY . .
 
-# Install dependencies and build
+### Install dependencies ###
 RUN node .scripts/update-parcelrc.js
 RUN pnpm i
+
+### Build ###
+# Admin Console build env
+ENV CONSOLE_PUBLIC_URL=/
```

Comment:
Need to add the `IS_CLOUD=true` to the env or add a `start:cloud` in the console project's `package.json` to specified the `IS_CLOUD` env.